build_posix/Make.base

```diff
@@ -6,10 +6,12 @@ ACLOCAL_AMFLAGS = -I build_posix/aclocal
 lib_LTLIBRARIES = libwiredtiger.la
 LDADD = $(lib_LTLIBRARIES)
 
+noinst_LTLIBRARIES = libpower8.la
```

Comment:
@daveh86 I'd rather this added a tag to the single file rather than creating a library.